<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <widget class="MyGLWidget" name="myGLWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>zRot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rotZSpinBox">
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="rotZSlider">
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>yRot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rotYSpinBox">
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="rotYSlider">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>xRot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rotXSpinBox">
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="rotXSlider">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="4">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0">
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>showOnLeft</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Height(cm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-200.000000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>CalGirth</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Girth(cm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header>myglwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setZRotation(int)</slot>
    <slot>setYRotation(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rotXSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setXRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotYSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setYRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotZSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setZRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
